packages/cactus-plugin-ledger-connector-besu/src/main/json/openapi.json

```diff
@@ -552,6 +552,35 @@
                         "nullable": false
                     }
                 }
+            },
+            "PrometheusExporterOptions": {
+                "type": "object",
+                "required": [
+                    "metricsPollingIntervalInMin"
+                ],
+                "properties": {
+                    "metricsPollingIntervalInMin": {
+                        "description": "The polling interval for the prometheus exporter (in minutes)",
+                        "type": "number",
+                        "nullable": false
+                    }
+                }
+            },
+            "PrometheusExporterMetricsRequest": {
+                "type": "object",
+                "required": [
+                    "promExporter"
+                ],
+                "properties": {
+                    "promExporter":{
```

Comment:
@jagpreetsinghsasan Is the `promExporter` being used?

Reply:
Fixed.